### ISSUE TITLE
Mohist SERVER ERROR

### ISSUE THREAD

**Issue by thestreetdj** [itzg/docker-minecraft-server#1344]
### Describe the problem

I'm trying to download and install the latest version (test) of the mohist server, and the docker stops working. (Synology Docker)

### Container definition

```
Paste run command or compose file here
```

### Container logs

<html>
<body>
<h2 style="text-align: center; color: rgb(0, 0, 0); font-family: &quot;Lucida Console&quot;, Consola, Monaco, monospace, &quot;Courier New&quot;, Courier, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">itzg-minecraft-server1</h2>

date | stream | content
-- | -- | --
2022-02-08 18:08:21 | stdout | [init]        and set VERSION accordingly
2022-02-08 18:08:21 | stdout | [init]        check https://ci.codemc.io/job/MohistMC/ for available versions
2022-02-08 18:08:21 | stdout | [init] ERROR: mohist builds do not exist for 1.18.1
2022-02-08 18:08:21 | stdout | [mc-image-helper] 18:08:21.067 WARN  : https://ci.codemc.io/job/MohistMC/job/Mohist-1.18.1/ cannot be retrieved: status=404
2022-02-08 18:08:18 | stdout | [init] Resolving type given mohist
2022-02-08 18:08:18 | stdout | [init] Resolved version given LATEST into 1.18.1
2022-02-08 18:08:15 | stdout | [init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 1 1000 1000 0 Feb  3 04:28 /data'

</body>
</html>
```

**Comment by itzg** [comment]
I haven't tried this out yet, but I think you need to set `VERSION` to exactly "1.18-dev"

https://ci.codemc.io/job/MohistMC/job/Mohist-1.18-dev/

**Comment by thestreetdj** [comment]
> 

Not only 1.18-dev but also 1.16.5 version has the same error.
This is Synology Docker.

**Comment by itzg** [comment]
Please provide your container definition

**Comment by thestreetdj** [comment]
> provide your container definition
[itzg-minecraft-server1.zip](https://github.com/itzg/docker-minecraft-server/files/8034086/itzg-minecraft-server1.zip)
Compress and upload the docker setting json file.

**Comment by itzg** [comment]
The output I got revealed more of the problem:

```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Feb  3 04:28 /data'
[init] Resolved version given 1.18-dev into 1.18-dev
[init] Resolving type given MOHIST
[init] ERROR: mohist build for 1.18-dev is not a valid server jar, found mohist-1.18.1-39.0.75-installer.jar
[init]        check https://ci.codemc.io/job/MohistMC/ for available versions
[init]        and set VERSION accordingly
```

The container's Mohist setup doesn't yet support the installer format.

**Comment by Mythik-Skaos** [comment]
I have the problem with Magma and Mohist in version 1.12.2
